src/setOfPeople.js

```diff
@@ -38,7 +38,19 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const final = new Set();
+
+  people.forEach(person => {
+    if (final.has(person)) {
+      final.delete(person);
+
+      return;
+    }
+
+    final.add(person);
```

Comment:
```suggestion
      final.delete(person);
    } else {
      final.add(person);
    }
```

Like this, it's getting a little shorter